From 32f4123eba2240ce85b096c6d71f5b7c6987fccf Mon Sep 17 00:00:00 2001
From: Miao Yu <r65093@freescale.com>
Date: Mon, 9 Mar 2009 15:28:06 +0800
Subject: [PATCH 079/435]     ENGR00109080 MX35: Add TVIN power-saving mode

    1. Support TVIN power-saving mode when system enter suspend/resume.
    After system enter suspend, adv7180 will enter power-saving mode;
    if wake up the system, tvin can resume from power-saving and work normally.
    2. Revise SENS_FRM_HEIGHT and  SENS_FRM_WIDTH register according to the
    reconstruction of V4L2 capture.
    (SENS_FRM_HEIGHT is equal to the sensor frame height - 1)
---
 test/mxc_v4l2_test/mxc_v4l2_tvin.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/mxc_v4l2_test/mxc_v4l2_tvin.c b/test/mxc_v4l2_test/mxc_v4l2_tvin.c
index b78c489..66d8492 100644
--- a/test/mxc_v4l2_test/mxc_v4l2_tvin.c
+++ b/test/mxc_v4l2_test/mxc_v4l2_tvin.c
@@ -235,7 +235,7 @@ main(int argc, char **argv)
         }
 
         /* Overlay setting */
-        alpha.alpha = 255;
+        alpha.alpha = 0;
 	alpha.enable = 1;
 	if ( ioctl(fd_fb, MXCFB_SET_GBL_ALPHA, &alpha) < 0) {
                 close(fd_fb);
-- 
1.8.0

